test/e2e/common_suite_test.go

```diff
@@ -52,6 +54,7 @@ type testCase struct {
 	testCommands         []testCommand
 	expectedPodLogString string
 	podState             v1.PodPhase
+	podWatcher           bool
```

Comment:
I wonder if imagePullTimer would be a better name for it based on the current implementation, or if you want a generically named `podWatcher` then move the image pull logging into `watchImagePullTime` and then pass it as a function parameter?